penny_university/settings/ci.py

```diff
@@ -1,4 +1,17 @@
+# from penny_university.settings.base import *
+#
+# # SECURITY WARNING: don't run with debug turned on in production!
+# DEBUG = True
+#
+# TASK_ALWAYS_EAGER = True
```

Comment:
Why are these commented out instead of deleted?

Reply:
I would expect that it would be always eager in CI for testing.

Reply:
silly me (they're undeleted/duplicated below this Corey)